configure.py

```diff
@@ -42,10 +42,80 @@
 from multiprocessing import Pool
 from subprocess import run
 
+_CPU_FAMILY_MAP = dict(
+    arm="arm",
+    aarch64="arm",
+    x86_64="x86",
+)
+
+_ANDROID_VERSION = 28
+
+_ANDROID_COMPILER_MAP = dict(
+    arm=f"armv7a-linux-androideabi{_ANDROID_VERSION}",
+    aarch64=f"aarch64-linux-android{_ANDROID_VERSION}",
+    x86_64=f"x86_64-linux-android{_ANDROID_VERSION}",
+)
+
+_ANDROID_ABI_MAP = dict(
+    arm="armeabi-v7a",
+    aarch64="arm64-v8a",
+    x86_64="x86_64",
+)
+
+_ANDROID_BUILD_MAP = dict(
+    Linux="linux-x86_64",
+    Darwin="darwin-x86_64",
+    Windows="windows-x86_64",
+)
+
+_ANDROID_CROSS_FILE_TPL = textwrap.dedent(
+    """\
+    [constants]
+    ndk_home = '{ndk_home}'
+    toolchain = ndk_home / 'toolchains/llvm/prebuilt/{ndk_build}'
+    [binaries]
+    c = toolchain / 'bin/{compiler}-clang'
+    cpp = toolchain / 'bin/{compiler}-clang++'
+    strip = toolchain / 'bin/llvm-strip'
+    ar = toolchain / 'bin/llvm-ar'
```

Comment:
could you add the `bin` directory to `toolchain` (or declare another constant)?

Reply:
Done.